src/engraving/dom/cmd.cpp

```diff
@@ -4030,7 +4030,7 @@ void Score::addRemoveBreaks(int interval, bool lock)
 
 void Score::cmdRemoveEmptyTrailingMeasures()
 {
-    auto beginMeasure = firstTrailingMeasure();
+    auto beginMeasure = masterScore()->firstTrailingMeasure();
```

Comment:
This will return a measure in the `MasterScore`, instead of in the current score. I wonder if `deleteMeasures` will work well if you feed it a measure from a different score.

Reply:
It seems, inside `deleteMeasures` they are used to get `Fraction`, so it works, but yes, You are right, it would be better, to do everything from masterScore.

Reply:
It seems, inside `deleteMeasures` measure parameters are used just to get `Fraction`, so it works, but yes, You are right, it would be better, to call everything from masterScore.